giskard/scanner/common/loss_based_detector.py

```diff
@@ -23,6 +23,9 @@ class LossBasedDetector(Detector):
 
     _needs_target = True
 
+    def __init__(self, max_dataset_size: int = MAX_DATASET_SIZE):
```

Comment:
inline this and probably divide by ~10?

Reply:
Ok, also maybe it still makes sense to divide by n_features if the param is not specified?

Reply:
yep, you could set `Optional[int] = None` and calculate the default at runtime if not provided